products/statement-generator/src/styles/appTheme.ts

```diff
@@ -16,6 +16,7 @@ const appTheme = createMuiTheme({
     },
     secondary: {
       main: '#FFFAF2', // tangerine light orange
+      dark: '#7422DA',
```

Comment:
Yeah, I see that the button is darker on the mockup than we have currently but instead of adding a slightly different purple let's just keep the [button color style](https://www.figma.com/file/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?node-id=2%3A541).